RELEASE.md

```diff
@@ -190,6 +190,20 @@ The release should be announced on the following social channels:
 - Twitter: Short and sweet in a single tweet, TODO: get Lodestar account.
 - Reddit: TODO: get Lodestar account.
 
+# Release Manager Checklist
+This section is to guide the Release Manager tasked with the next version release to ensure all items have been completed.
+- Confirm consensus on `unstable` release candidate commit
+- Complete Step 1: Create release candidate
+- Complete Step 2: Tag release candidate
+- Deploy `rc.x` candidate to `beta` group of servers
```

Comment:
I think we should start thread on communication channels from this step so that everybody gets informed about the status of the release